_workinggroups/quantumcomputing.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Quantum Computing"
+author: "Riccardo Maria Bianchi"
+layout: plain
+---
+
+
+# The HSF Quantum Computing working group
+
+## Introduction
+
+The Quantum Computing Working Group fosters collaboration among the HEP experiments and institutes to exploit the possibilities offered by quantum computing.
+
+Quantum computing is a very active research field and prototype computers, simulators, and toolkits are now available. The participants in the working group share experiences, ideas, know-how, demonstrators, code snippets, and documents with the aim of fostering common research and development in this field, for the benefit of the HEP community.
+
+Kick-starters: *Riccardo Maria BIANCHI* (ATLAS), *Sebastien BINET* (LSST, ALICE, LPC), *Riccardo DI SIPIO* (ATLAS), *Lukas HEINRICH* (ATLAS), *Maurizio PIERINI* (CMS)
```

Comment:
Contact info is needed - one way to contact the convenors and one way to contact the group. I think the group email list exists, right? For the convenors it's probably best that you setup a Google group with the 5 of you on it.

Reply:
Done, and added to the QC page.